package.json

```diff
@@ -52,6 +52,7 @@
     "query-string": "~6.11.1",
     "react": "~16.13.0",
     "react-dom": "~16.13.0",
+    "react-draggable": "^4.2.0",
```

Comment:
please manually replace `^` with tilde (`~`) in dependecies, that makes them much more stable and reliable

Reply:
And don't forget to update `yarn.lock` afterwards